packages/react/src/components/HorizontalRule/HorizontalRule.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import settings from 'carbon-components/es/globals/js/settings';
+
+import '@ibmdotcom/styles/scss/components/horizontalrule/_horizontalrule.scss';
```

Comment:
@annawen1 one more thing, we are keeping style imports separate from the JS. Can you import this in the story instead?

Reply:
yep done